README.md

```diff
@@ -60,3 +60,7 @@ See our contribution guidelines in these languages:
 - [Indonesian](CONTRIBUTING_ID.md)
 
 When contributing to our project, please use English when communicating with other people in issues and/or pull requests. [Click here](CONTRIBUTING.md#why-are-we-using-english-in-our-issues--prs) to read why. ([Bahasa Indonesia](CONTRIBUTING_ID.md#mengapa-kita-menggunakan-bahasa-inggris-dalam-menulis-issue-dan-pull-request))
+
+## API
+
+for api documentation, [click here](docs/api-reference.md)
```

Comment:
```suggestion
Our API documentation is in [api-reference.md](docs/api-reference.md) file.
```

Rephrase the sentence to make it more obvious.

Reply:
Done, please check it agian.